che-incubator/che-code
# fix: Fix displaying Copy endpoint URL to clipboard icon

### What does this PR do?
Fixes displaying Copy endpoint URL to clipboard icon

### What issues does this PR fix?

https://github.com/eclipse/che/issues/22512

### How to test this PR?
Hover over the endpoint actions panel and check that the icon is displayed

![image](https://github.com/che-incubator/che-code/assets/5676062/3cbf6809-19da-4fe8-8d02-57989935a1b5)

![image](https://github.com/che-incubator/che-code/assets/5676062/a6204cac-3af6-4d99-a507-ff25aa445e4e)
